libcob/termio.c

```diff
@@ -225,9 +225,12 @@ clean_double (char *wrk)
 	char *pos = strrchr (wrk, 'E');
 
 	if (pos) {
+		char *src;
 		pos += 2; /* skip E+ */
-		if (pos[0] == '0') {
-			memmove (pos, pos + 1, strlen (pos));
+		/* skip leading zeroes */
```

Comment:
please add a comment that each COBOL environment do display them in a different way and we ensure here for internal consistency (and if there's enough request we may add different output formats later)